### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by therealrambo**
removed redundant line on the f-droid inclusion policy.

- [x] I carefully read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md) and agree to them.

### CODE REVIEW

**Comment by gkeegan** [comment]
I think both lines there are important, and you just removed a valuable link for people to discover what is meant.

**Comment by therealrambo** [comment]
there is no valuable link removed. both lines link to the same page and say "use this guideline with your code." there is nothing of importance added on the second one.

**Comment by gkeegan** [comment]
Oh I see I thought you meant a different line is the copy. 

Perhaps remove the line "stick to the guidelines" and keep the one you are removing?

**Comment by therealrambo** [comment]
maybe, it does not really matter. the second one has some grammar issues though.

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -52,7 +52,6 @@ with your GitHub account.
   the core developers, and just adds work. If you do not intend to maintain features you contributed, please think again
   about submission, or clearly state that in the description of your PR.
 * Respond yourselves if someone requests changes or otherwise raises issues about your PRs.
-* Check if your contributions align with the [fdroid inclusion guidelines](https://f-droid.org/en/docs/Inclusion_Policy/).
 * Check if your submission can be build with the current fdroid build server setup.
```
Comment (B0pol) [inline .github/CONTRIBUTING.md]: It should be F-Droid, not fdroid

**Comment by therealrambo** [comment]
I'm sorry if I didn't make this clear, the sentence I removed is almost exactly like an earlier sentence in this section; that sentence is "Stick to F-Droid contribution guidelines." They also both have the same link to the same website.

**Comment by Stypox** [comment]
@therealrambo while you are at it, could you please change all occourences of "fdroid" with "F-Droid"? Sorry for the late reply

**Pull request closed without merging**